projects/python-client/tests/resources/sample_parameters.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
I'm assuming this didn't detect as a Move due to pre-commit adding whitespace.

Rest assured this isn't a new file (you can see it being removed at the end).